src/main/resources/hudson/scm/cvstagging/CvsTagAction/tagForm.jelly

```diff
@@ -87,9 +87,9 @@ THE SOFTWARE.
           </p>
 
           <j:if test="${h.hasPermission(it.build,it.permission)}">
+            <st:adjunct includes="hudson.scm.cvstagging.CvsTagAction.create-another-tag"/>
             <p id="tagButton">
-              <input type="button" value="${%Create another tag}"
-                     onclick="Element.show('tagForm');Element.hide('tagButton')"/>
+              <input type="button" value="${%Create another tag}" id="create-another-tag-button" />
```

Comment:
No need for a new ID I think. You can just use the existing ID for the parent element `tagButton` and select for an `input` child (of which there is only one in the markup) with `#tagButton input`. Simple and avoids cluttering the DOM with yet another ID that is only used once.